src/main/java/subaraki/hangman/entity/HangEntityDummy.java

```diff
@@ -102,10 +102,10 @@ public double getPassengersRidingOffset() {
         if (!this.getPassengers().isEmpty()) {
             Entity e = this.getPassengers().get(0);
             if (!(e instanceof Player))
-                return -e.getEyeHeight();
+                return -e.getEyeHeight() + 0.35;
         }
 
-        return -1.37D;
+        return -1.42D;
```

Comment:
Looks better in my opinion.
Also, makes the noose look the same on both players and zombies.